server.test.js

```diff
@@ -0,0 +1,135 @@
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const mongoose = require('mongoose');
+const request = require('supertest');
+const path = require('path');
+const fs = require('fs');
+const Neighborhood = require('./database/models/neighborhoods.js');
+
+
+const mockNeighborhood = {
+  name: 'Delia',
+  city: 'Shangchewan',
+  country: 'China',
+  gettingAround: 'Cras non velit nec nisi vulputate nonummy. Maecenas tincidunt lacus at velit. Vivamus vel nulla eget eros elementum pellentesque.',
+  description: 'Nullam porttitor lacus at turpis. Donec posuere metus vitae ipsum. Aliquam non mauris.',
+  latitude: 29.806827,
+  longitude: 113.013638,
+  guidebookId: 2
```

Comment:
leave a comma - even though this is the end of your object, our styleguide requires us to leave "trailing commas"

Reply:
(line 17)